tests/e2e/cli-before-regional-blobs-support.test.ts

```diff
@@ -18,8 +19,17 @@ test('should serve 404 page when requesting non existing page (no matching route
 
   expect(await page.textContent('h1')).toBe('404')
 
+  // https://github.com/vercel/next.js/pull/69802 made changes to returned cache-control header,
+  // after that (14.2.10 and canary.147) 404 pages would have `private` directive, before that it
+  // would not
+  const shouldHavePrivateDirective = nextVersionSatisfies(
+    '>=14.2.10 <15.0.0 || >=15.0.0-canary.147',
+  )
   expect(headers['netlify-cdn-cache-control']).toBe(
-    'no-cache, no-store, max-age=0, must-revalidate, durable',
+    (shouldHavePrivateDirective ? 'private, ' : '') +
+      'private, no-cache, no-store, max-age=0, must-revalidate, durable',
```

Comment:
We conditionally adding `private`, before that check
```suggestion
      'no-cache, no-store, max-age=0, must-revalidate, durable',
```